README.md

```diff
@@ -243,6 +243,9 @@ This is a custom hook that takes care of fetching your query and storing the res
     - `previousData`: Previous GraphQL query or `updateData` result
     - `data`: New GraphQL query result
   - `client`: GraphQLClient - If a GraphQLClient is explicitly passed as an option, then it will be used instead of the client from the `ClientContext`.
+  - `refetchAfterMutations`: Array of RefetchAferMutationsData - Mutations that will cause the refetch of the query
+    - `mutation`: String - The mutation string
+    - `filter`: Function (optional) - It receives mutation variables as parameter and blocks refetch if it returns false
```

Comment:
let's make this option "dynamic" to be more user friendly and accept:

- string
- object
- array of {string | object}

the code will have to change to handle those cases